<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="emailback">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>347</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 345px;
height: 60px;
border-radius: 6px;
border: 2px solid #cadee7;
background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="emailback_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>347</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 345px;
height: 60px;
border-radius: 6px;
border: 2px solid #cadee7;
background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="emailback_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>347</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 345px;
height: 60px;
border-radius: 6px;
border: 2px solid #cadee7;
background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="emailback_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>347</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 345px;
height: 60px;
border-radius: 6px;
border: 2px solid #cadee7;
background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>290</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 345px;
height: 60px;
border-radius: 6px;
border: 0px solid #cadee7;

background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="surname">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>290</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 345px;
height: 60px;
border-radius: 6px;
border: 0px solid #cadee7;

background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
   <property name="placeholderText">
    <string>Surname</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>250</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 345px;
height: 60px;
border-radius: 6px;
border: 0px solid #cadee7;

background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
   <property name="placeholderText">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 345px;
height: 60px;
border-radius: 6px;
border: 0px solid #cadee7;

background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 60px;
height: 60px;

background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="password_icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 60px;
height: 60px;

background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="user_icon_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 60px;
height: 60px;

background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="user_icon_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 60px;
height: 60px;

background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>500</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_2 {
	background-image:/img/Rectangle_4.png;
width: 345px;
height: 60px;
border-radius: 12px;
background-color: #7cee81;


color: #ffffff;
font-family: &quot;SF UI Display&quot;;
font-size: 18px;
font-weight: 600;
line-height: 20px;
/* Text style for &quot;Sign up&quot; */
letter-spacing: 0.5px;
}

#pushButton_2:hover {
   background: #23D035;
   background-image: -webkit-linear-gradient(top, #23D035, #3D94F6);
   background-image: -moz-linear-gradient(top, #23D035, #3D94F6);
   background-image: -ms-linear-gradient(top, #23D035, #3D94F6);
   background-image: -o-linear-gradient(top, #23D035, #3D94F6);
   background-image: linear-gradient(to bottom, #23D035, #3D94F6);
   text-decoration: none;
}
#pushButton_2:active {
	position:relative;
	top:1px;
}</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 300px;
height: 96px;
color: #85b9ca;
font-family: &quot;SFUI Display - Heavy&quot;;
font-size: 25px;
font-weight: 800;
/* Text style for &quot;Let’s get&quot; */
letter-spacing: 0.6px;</string>
   </property>
   <property name="text">
    <string>Let’s get you
started</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>111</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="photo">
      <property name="styleSheet">
       <string notr="true">width: 64px;
height: 64px;
background-color: #e1e1e1;
-webkit-border-radius: 50px;
-moz-border-radius: 50px;
border-radius: 50px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="upload">
      <property name="styleSheet">
       <string notr="true">#upload{
width: 101px;
height: 48px;
border-radius: 4px;
background-color: #ffa726;

color: #ffffff;
font-family: &quot;SF UI Display&quot;;
font-size: 18px;
font-weight: 600;
line-height: 20px;
}
#upload:hover {
   background: #D0D03C;
   background-image: -webkit-linear-gradient(top, #D0D03C, #3D94F6);
   background-image: -moz-linear-gradient(top, #D0D03C, #3D94F6);
   background-image: -ms-linear-gradient(top, #D0D03C, #3D94F6);
   background-image: -o-linear-gradient(top, #D0D03C, #3D94F6);
   background-image: linear-gradient(to bottom, #D0D03C, #3D94F6);
   text-decoration: none;
}
#upload:active {
	position:relative;
	top:1px;
}</string>
      </property>
      <property name="text">
       <string>UPLOAD</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>emailback</zorder>
  <zorder>user_icon</zorder>
  <zorder>emailback_2</zorder>
  <zorder>emailback_3</zorder>
  <zorder>emailback_4</zorder>
  <zorder>name</zorder>
  <zorder>surname</zorder>
  <zorder>user_icon_2</zorder>
  <zorder>email</zorder>
  <zorder>pushButton</zorder>
  <zorder>user_icon_3</zorder>
  <zorder>password_icon</zorder>
  <zorder>password</zorder>
  <zorder>status</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
